### DISCUSSION TITLE
Is it possible to add "Industry" field to the OSCP?

### DISCUSSION THREAD

**Discussion by SherpaCLT** [bullhorn/career-portal#389] [Q&A] [answered]
Linked In scrapes jobs from our career portal - when this is done, they are assigning all of our jobs to our Industry (Recruiting and Staffing" instead of those of our clients.  Because their system serves job ads based in part on that criteria, we're finding it isn't delivering as well as it could.

Would it be easy to pull the "Industry" field from BH into the OSCP, so they could scrape it and properly categorize the jobs?  I don't really care where it shows up (maybe beside "Category").

Any ideas?

**Comment by charlesabarnes** [comment, marked as answer]
Hello,

This is possible but would require some modification to your portal.  Industry (or `businessSectors`) has a "to-many" association with Joborder.  To-many associations are not currently supported to display as "job-chips" natively, so your portal would need to be modified to loop over every option to display it.

Alternatively, another field can be leveraged reflect "Industry" as well and would have a more straightforward implementation

**Comment by SherpaCLT** [comment]
Thanks for that Charles.

I just downloaded the latest version of the OSCP, but I have to admit, aside from changing the colors and the auto response message I’ve never made any coding changes to any version of the OSCP (coding novice here for sure).

It sounds like the second option you mention might be easier to accomplish.  In that case, would I essentially create a field in each JO’s field mappings that mirrors the options in the businessSectors field?  If I did that, could I then create a job-chip showing the proper industry for each opportunity?

Thanks!

Greg Whitesell | Marketing Director | 704.374.0001

[Sherpa Email Logo 2019]

**Reply by charlesabarnes** [reply]
Exactly!  After that is done, you would need to contact Bullhorn Support to "expose" the field to public API.  Then, in the `app.json` file, you will want to add the field to `fields` in the app.json file, and to `jobInfoChips`
